dotCMS/src/main/java/com/dotcms/experiments/business/result/ExperimentResults.java

```diff
@@ -46,18 +50,25 @@ public TotalSession getSessions() {
     /**
      * Return the result split by {@link com.dotcms.variant.model.Variant}.
      *
-     *
      * @return
      */
     public Map<String, GoalResults> getGoals() {
         return goals;
     }
 
+    public BayesianResult getBayesianResult() {
```

Comment:
this is going to change what the end point response so I think we need to change the postman test here, are you sure that we need to return all the BayesianResult Object? is it not too much because maybe we just need to return the result.

Also we are not returning if we have a winner

Reply:
It won't hurt to have all this result data available in the client side.